### ISSUE TITLE
Use strip_namespaces with .to_dict()?

### ISSUE THREAD

**Issue by kevbutler** [sissaschool/xmlschema#161]
I'm using xmlschema to both validate and decode several documents into Python dictionaries. Similar to #113 , I'd like to remove the namespace prefixes from the tags, but am not sure how to do this with `.to_dict()`.

My original code follows, and it works without issue when I leave namespaces in the results:
```python
schema = xmlschema.XMLSchema('xsd_file.xsd', validation='strict')
parsed =  schema.to_dict(xml_doc, validation='lax')
```

I've tried incorporating `strip_namespaces` like this:
```python
converter = xmlschema.XMLSchemaConverter(strip_namespaces=True)
schema = xmlschema.XMLSchema('xsd_file.xsd', validation='strict', converter=converter)
parsed =  schema.to_dict(xml_doc, validation='lax')
```
but this results in `TypeError: local_name() takes 1 positional argument but 2 were given`. Is there something else that I'm missing?

Here's the traceback:

```
<ipython-input-729-1ddd08197b50> in parse_xml(path, doc_num, doc, schema)
    170 def parse_xml(path, doc_num, doc, schema):
    171     # Validate, parse, and convert XML to dict
--> 172     parsed =  schema.to_dict(doc, validation='lax')
    173 
    174     # Add file info

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/validators/schema.py in decode(self, source, path, schema_path, validation, *args, **kwargs)
   1430         """
   1431         data, errors = [], []
-> 1432         for result in self.iter_decode(source, path, schema_path, validation, *args, **kwargs):
   1433             if not isinstance(result, XMLSchemaValidationError):
   1434                 data.append(result)

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/validators/schema.py in iter_decode(self, source, path, schema_path, validation, process_namespaces, namespaces, use_defaults, decimal_type, datetime_types, converter, filler, fill_missing, max_depth, **kwargs)
   1415                     return
   1416 
-> 1417             for obj in xsd_element.iter_decode(elem, validation, **kwargs):
   1418                 yield obj
   1419 

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/validators/elements.py in iter_decode(self, elem, validation, **kwargs)
    653         if converter is not None:
    654             element_data = ElementData(elem.tag, value, content, attributes)
--> 655             yield converter.element_decode(element_data, self, level)
    656         if content is not None:
    657             del content

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/converters.py in element_decode(self, data, xsd_element, level)
    280         else:
    281             if data.attributes:
--> 282                 result_dict.update(t for t in self.map_attributes(data.attributes))
    283 
    284             has_single_group = xsd_element.type.content_type.is_single()

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/converters.py in <genexpr>(.0)
    280         else:
    281             if data.attributes:
--> 282                 result_dict.update(t for t in self.map_attributes(data.attributes))
    283 
    284             has_single_group = xsd_element.type.content_type.is_single()

~/.pyenv/versions/3.7.6/envs/xml-test/lib/python3.7/site-packages/xmlschema/converters.py in map_attributes(self, attributes)
    192         elif self.attr_prefix:
    193             for name, value in attributes:
--> 194                 yield '%s%s' % (self.attr_prefix, self.map_qname(name)), value
    195         else:
    196             for name, value in attributes:

TypeError: local_name() takes 1 positional argument but 2 were given
```

I appreciate any help!

**Comment by brunato** [comment]
Hi,
sorry but this is a bug that happens when `strip_namespace=True` is used. 
I've moved *strip_namespace* to base class *NamespaceMapper* in order to define *map_qname* and *unmap_qname* as aliases for protected methods. These aliases are reassigned to a local name mapper method when strip_namespace=True is provided.

The fix will be available with the release v1.1.0 soon.
Thanks

**Comment by brunato** [comment]
Fix available in v1.1.0.
Bye, 
Davide
